artifacts/camara-cloudevents/subscription-notification-template.yaml

```diff
@@ -0,0 +1,759 @@
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SubscriptionAsync'
+        "400":
+          $ref: '#/components/responses/SubscriptionIdRequired'
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/Generic404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+components:
+  securitySchemes:
+    openId:
+      type: openIdConnect
+      openIdConnectUrl: https://example.com/.well-known/openid-configuration
+  parameters:
+    SubscriptionId:
+      name: subscriptionId
+      in: path
+      description: Subscription identifier that was obtained from the create event subscription operation
+      required: true
+      schema:
+        $ref: '#/components/schemas/SubscriptionId'
+    x-correlator:
+      name: x-correlator
+      in: header
+      description: Correlation id for the different services
+      schema:
+        type: string
+  headers:
+    x-correlator:
+      description: Correlation id for the different services
+      schema:
+        type: string
+  schemas:
+    ErrorInfo:
+      type: object
+      required:
+        - status
+        - code
+        - message
+      properties:
+        status:
+          type: integer
+          description: HTTP response status code
+        code:
+          type: string
+          description: Code given to this error
+        message:
+          type: string
+          description: Detailed error description
+        
+    SubscriptionRequest:
+      description: The request for creating a xxx event subscription
+      type: object
+      required:
+        - sink
+        - protocol
+        - config
+      properties:
+        protocol:
+          $ref: "#/components/schemas/Protocol"
+        sink:
+          type: string
+          format: url
+          description: The address to which events shall be delivered using the selected protocol.
+          example: "https://endpoint.example.com/webhook"
+        sinkCredential:
+          $ref: '#/components/schemas/SinkCredential'
+        types:
+          description: Camara Event types eligible to be delivered by this subscription - Use of array MUST be decided at API project level.
+          type: array
+          items:
+            type: string
+        filters:
+          type: object
+          description: Placeholder to add filters management for futur UC. **Not to be used as of now.**
+        config:
+          $ref: '#/components/schemas/Config'
+      discriminator:
+        propertyName: protocol
+
+    Protocol:
+      type: string
+      enum: ["HTTP", "MQTT3", "MQTT5", "AMQP", "NATS", "KAFKA"]
```

Comment:
Do we need other protocols in the array and the related schema definitions (like MQTTSettings) ?

Reply:
Fair question; I noted to our last week discussion that we align on this point with CloudsEvents subscriptions but only accommodate use of HTTP as of now.
Probably we can keep it as such in the template but

- only document use of HTTP in the guideline
- recommend to project to only have HTTP in their yaml

WDYT?

Reply:
I prefer to have only HTTP in the template. It is the unique case we have so far.
 
 @bigludo7, @shilpa-padgaonkar if you prefer to keep it for extensibility in future. Then at least to document in design guidelines only HTTP is supported and also indicate to adapt to taht value when applied in a specific WG

Reply:
We can surely document that this is currently restricted to just HTTP.

Reply:
Ludovic, has added an exception. It is ok to me.

Reply:
resolving this. Exception indicated